Sources/NIOHTTP2/HTTP2ChannelHandler+InboundStreamMultiplexer.swift

```diff
@@ -135,6 +135,15 @@ extension NIOHTTP2Handler.InboundStreamMultiplexer {
             break // do nothing
         }
     }
+
+    func processedFrame(streamID: HTTP2StreamID, size: Int) {
```

Comment:
I think this spelling is a bit weird, and it feels like a bit of a layering violation to me. I'm inclined to say we should pass the entire frame in here, and only do the size computation once we get inside the `inlineStreamMultiplexer`.